modules/gitops-creating-user-defined-cluster-roles-and-configuring-user-defined-permissions-for-application-controller.adoc

```diff
@@ -0,0 +1,201 @@
+// Module included in the following assembly:
+//
+// * declarative_clusterconfig/customizing-permissions-by-creating-aggregated-cluster-roles.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-creating-user-defined-cluster-roles-and-configuring-user-defined-permissions-for-application-controller_{context}"]
+= Creating user-defined cluster roles and configuring user-defined permissions for Application Controller
+
+As a cluster administrator, to add user-defined permissions to your aggregated cluster role, you must create one or more user-defined cluster roles and configure user-defined permissions for Argo CD Application Controller of a cluster-scoped Argo CD instance.
+
+.Prerequisites
+
+* You have enabled the creation of aggregated cluster roles for the Argo CD Application Controller component of a cluster-scoped Argo CD instance.
+* You have the following default cluster roles that are created and managed by the {gitops-title} Operator:
++
+** `<argocd_name>-<argocd_namespace>-argocd-application-controller` aggregated cluster role
```

Comment:
Suggestion: The phrase "aggregate cluster role" that isn't included in the backtick looks out of place as you have followed it for a single bullet point. I feel we should either use a similar template for the remaining bullets or improve the existing bullet.

Reply:
Sorted the conversation with @Dhruv-Soni11. Retaining the content as is.